apps/meteor/tests/e2e/omnichannel/omnichannel-triggers.spec.ts

```diff
@@ -19,55 +19,134 @@ test.describe.serial('omnichannel-triggers', () => {
 			email: faker.internet.email(),
 		};
 		triggersName = faker.string.uuid();
-		triggerMessage = 'Welcome to Rocket.chat';
-		await Promise.all([
+		triggerMessage = 'Welcome to Rocket Chat';
```

Comment:
I think the company's name and usually every place we mention we mention as Rocket.Chat :x